gapic/src/main/com/google/gapid/models/Capture.java

```diff
@@ -50,6 +50,10 @@
 public class Capture extends ModelBase<Capture.Data, File, Loadable.Message, Capture.Listener> {
   protected static final Logger LOG = Logger.getLogger(Capture.class.getName());
 
+  // Don't try to open files with 16 or less bytes. An empty graphics trace, without the
+  // capture header is already 16 bytes.
```

Comment:
"is already 16 bytes" -> is already *more than* 16 bytes?

Reply:
They are exactly 16bytes.